src/containers/search/resultsList.js

```diff
@@ -51,13 +53,36 @@ class ResultsList extends Component {
     );
   }
 
+  renderRelatedData(d) {
+    if (!d.relatedData || d.relatedData.length === 0 || d.relatedData[0] === '') { return ''; }
+
+    const _links = d.relatedData.map( x => {
+      let queryParams = {category: x.category};
+      queryParams[x.targetField] = x.sourceName;
+      const categoryLabel = CATEGORIES.filter( cat => cat.name === x.category )[0].displayName;
```

Comment:
If you just want the first match anyway: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find